src/App/App.js

```diff
@@ -20,6 +20,12 @@ const App = () => {
     "100000020",
     "100000021",
   ];
+  return cardlist.sort(() => 0.5 - Math.random());
+}
+
+const App = () => {
+
+  const cardlist = generateCardList();
```

Comment:
```suggestion
  const [cardlist, setCardlist] = useState([]);
  
  handleRestart = () => {
    setCardlist(generateCardList())
  }
```

Detalhe: pra usar o useState você precisa adicionar esse import lá no começo do arquivo

```js
import { useState } from 'react'
```

Reply:
Faltou um const antes do handleRestart, já fiz o commit com ele.

Reply:
Vamos marcar um reunião no Whereby, dai você me explica para que serve, por enquanto, se aparecer um outro caso parecido ou usar como tá ai. :)